src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -177,10 +180,6 @@ private void handleExit() {
         primaryStage.hide();
     }
 
-    public EmployeeListPanel getEmployeeListPanel() {
```

Comment:
I'm not too familiar with the UI -- is it safe to remove @starrylight99 ?

Reply:
It is a relic method, and was used in a very [old version](https://github.com/AY2324S1-CS2103-T16-1/tp/commit/b8ed0900daea2e5e2f771cd4453099a9eaa6ce19#diff-b22ae445aa16b18108001a435997f5c179c75c886955543c0caf13077d2c5d38) of the UI. It is now obsolete.

Reply:
Ok ok!